specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -0,0 +1,2663 @@
+    },
+    "produces": [
+     "application/json"
+    ]
+   },
+   "delete": {
+    "description": "Delete an existing face list according to faceListId. Persisted face images in the face list will also be deleted.",
+    "operationId": "FaceList_Delete",
+    "parameters": [
+     {
+      "name": "faceListId",
+      "in": "path",
+      "description": "Id referencing a Face List.",
+      "required": true,
+      "type": "string",
+      "pattern": "^[a-z0-9-_]+$",
+      "maxLength": 64
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns an empty response body."
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  },
+  "/facelists": {
+   "get": {
+    "description": "Retrieve information about all existing face lists. Only faceListId, name and userData will be returned.",
+    "operationId": "FaceList_List",
+    "parameters": [
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns an array of faceList.",
+      "schema": {
+       "$ref": "#/definitions/FaceListResults"
+      },
+      "examples": {
+       "application/json": [
+        {
+         "faceListId": "sample_list",
+         "name": "list1",
+         "userData": "User-provideddataattachedtothefacelist"
+        }
+       ]
+      }
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  },
+  "/facelists/{faceListId}/persistedFaces/{persistedFaceId}": {
+   "delete": {
+    "description": "Delete an existing face from a face list (given by a persisitedFaceId and a faceListId). Persisted image related to the face will also be deleted.",
+    "operationId": "FaceList_DeleteFace",
+    "parameters": [
+     {
+      "name": "faceListId",
+      "in": "path",
+      "description": "faceListId of an existing face list.",
+      "required": true,
+      "type": "string",
+      "maxLength": 64,
+      "pattern": "^[a-z0-9-_]+$"
+     },
+     {
+      "name": "persistedFaceId",
+      "in": "path",
+      "description": "persistedFaceId of an existing face. ",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns an empty response body."
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  }
+ },
+ "x-ms-paths": {
+  "/detect?overload=url": {
```

Comment:
Quick Question -- does the service expect the `overload=url` parameter to actually be sent? or does it infer the operation from the post contents?